### DISCUSSION TITLE
Submitting message to Assistant

### DISCUSSION THREAD

**Discussion by tonilampela** [zed-industries/zed#10668] [Feature Requests]
I must be doing something wrong...

I'm testing Assistant functionality for the first time in Zed and I can't seem to get submitting messages to Assistant work using "⌘ + enter" key combo. It doesn't do anything, and only way to workaround is open command palette (⌘ + shift + p) and select "Assist" option (which also shows "⌘ + enter" as shortcut). 

I am in VIM mode, but I guess that shouldn't matter as submit shortcut doesn't work either in interactive or non-interactive mode.

Edit: or is there some VIM mode specific shortcut to submit? If there is, would be nice to have it documented.

**Comment by xpe** [comment]
@tonilampela Have you confirmed that your API key works by interacting with https://platform.openai.com ? It is possible you don't have any credits in your account.

Dear Zed team: The Zed assistant pane doesn't seem to show errors such as (roughly) "not enough credits". I'll check the issue tracker to see if you are aware of this problem.

**Comment by KaraniAbdellah** [comment]
My Question: why Zed editor does not support .http filies ?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
